shared/AppInsightsCore/src/JavaScriptSDK/AppInsightsCore.ts

```diff
@@ -348,130 +348,18 @@ export class AppInsightsCore<CfgType extends IConfiguration = IConfiguration> im
                 // This will be "re-run" if the referenced config properties are changed
                 _addUnloadHook(_configHandler.watch((details) => {
                     let rootCfg = details.cfg;
-                  
-                    let isPending = _activeStatus === eActiveStatus.PENDING;
-                    
-                    if (isPending){
```

Comment:
Moved this logic around as by having this here this will cause subsequent "calls" for the onConfigChanges to not tag / reference any contained properties which would then result on the onConfigChange to never get called again (as the last run didn't reference anything)

Found this as part of adding new configuration options for the OTel beta as I was adding additional checks.